NightscoutServiceKit/OTPManager.swift

```diff
@@ -147,10 +155,10 @@ public class OTPManager {
         return Token(name: secretKeyName, issuer: issuerName, generator: generator)
     }
     
```

Comment:
Return nil if we can't get the token rather than just returning an empty array to avoid ambiguity. An error may be preferred but this seems like an improvement.